### PR TITLE
docs: rfc-0699: push notifications apns

### PR DESCRIPTION
**Pull request by berendsliedrecht**
- Rfc that enables agents to exhange a push notification configuration 
- It is possible with the discover features protocols, the roles specifically, to check which notification services another agent can provide.

Interested to hear any feedback!

Signed-off-by: Berend Sliedrecht <berend@animo.id>

### CODE REVIEW

**Comment by TimoGlastra** [comment]
PR with client side implementation in AFJ: https://github.com/hyperledger/aries-framework-javascript-ext/pull/41

File: features/0699-push-notifications/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+### Sending Push Notifications
+
+When an agent wants to send a push notification to another agent, the payload of the push notifications MUST include the `@type` property to indicate the message is sent by the notification-sender. Guidelines on notification messages are not defined.
+
+```jsonc
+{
+  "@type": "https://didcomm.org/push-notifications",
+  ...
+}
```
Comment (TimoGlastra) [inline features/0699-push-notifications/README.md:155]: Should we include a message_tag (or message_tag_list) in the push notification to indicate which messages it is about.

**Comment by TelegramSam** [comment]
Discussed 20211215. Would prefer a different protocol for each push notification method to simplify discovery and impl of each protocol. This addresses both Android and iOS, and they should be split.

**Comment by blu3beri** [comment]
@TelegramSam It took a while, but I have resolved the feedback from the conversation of a while back. I will soon raise a PR with almost the same RFC, but for Android.

**Comment by TimoGlastra** [comment]
Discussed during 20220518 WG call. Some points to address (for both RFCs):
- make fcm-android just fcm
- in addition to the message_tag for push notification data also add a message_id (as used in the pickup protocol v2)

**Pull request merged**